### PR TITLE
Beam 2.47.0 Release Website Updates

### PR DESCRIPTION
**Pull request by jrmccluskey**
Updates the website with the release listing and blog post

DO NOT SUBMIT until an 2.47.0 RC candidate is approved and the remaining fields are updated. 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by jrmccluskey** [comment]
Run Website_Stage_GCS PreCommit

**Comment by jrmccluskey** [comment]
R: @damccorm

**Review by damccorm — APPROVED** [review]
Only comment is the archive urls, otherwise LGTM

File: website/www/site/content/en/get-started/downloads.md [hunk 1]
```diff
@@ -96,6 +96,14 @@ versions denoted `0.x.y`.
...
+### 2.47.0 (2023-05-10)
+Official [source code download](https://downloads.apache.org/beam/2.47.0/apache-beam-2.47.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.47.0/apache-beam-2.47.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.47.0/apache-beam-2.47.0-source-release.zip.asc).
+
+[Release notes](https://github.com/apache/beam/releases/tag/v2.47.0)
+[Blog post](/blog/beam-2.47.0).
+
 ### 2.46.0 (2023-03-10)
 Official [source code download](https://downloads.apache.org/beam/2.46.0/apache-beam-2.46.0-source-release.zip).
```
[the remark this answers is not in the document]
Comment (jrmccluskey) [reply website/www/site/content/en/get-started/downloads.md]: Updated

**Pull request merged**